saleor/graphql/plugins/dataloaders.py

```diff
@@ -46,6 +47,12 @@ def batch_load(self, keys):
 def plugin_manager_promise(context: SaleorContext, app) -> Promise[PluginsManager]:
     user = context.user
     requestor = app or user
+
+    if isinstance(requestor, SimpleLazyObject):
+        if requestor._wrapped is empty:  # type: ignore[attr-defined]
```

Comment:
Reuse this function?

https://github.com/saleor/saleor/blob/2e261eae739660da427e5f40e1efc6e19639dc1b/saleor/core/utils/lazyobjects.py#L35-L39